src/model/alerts.js

```diff
@@ -1,4 +1,5 @@
 import { Schema } from "mongoose";
+import { connectionAPI } from "../config";
```

Comment:
You can import multiple variables from the same file, lets add this on one line so that we dont have to duplicate the import of the config
https://github.com/jembi/openhim-core-js/pull/899/commits/38f9164829f0eb268b7e55ed12d879271af780ae#diff-9c101532f3045298b17eb1c108b13891R4